### PR TITLE
fixed bugs 792 805 861

### PR DESCRIPTION
**Pull request by sofiiamatsyhin**

### CODE REVIEW

File: src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.scss [hunk 1]
```diff
@@ -169,11 +183,20 @@ div {
...
+        // .user-data-text-date:nth-child(2) {
+        //   max-height: 26px;
+        //   display: block;
+        //   white-space: nowrap;
+        //   max-width: 110px;
+        //   overflow: hidden;
+        //   text-overflow: ellipsis;
+        // }
       }
     }
```
Comment (Gusvarov) [inline src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.scss:201]: remove comments

File: src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.scss [hunk 2]
```diff
@@ -203,12 +226,20 @@ div {
...
+      // .user-data-text-date:nth-child(2) {
+      //   max-height: 26px;
+      //   display: block;
+      //   white-space: nowrap;
+      //   max-width: 110px;
+      //   overflow: hidden;
+      //   text-overflow: ellipsis;
+      // }
     }
   }
```
Comment (Gusvarov) [inline src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.scss:244]: same as 192

Comment (sofiiamatsyhin) [reply src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.scss]: fixed

File: src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.scss (hunk 1 above)
Comment (sofiiamatsyhin) [reply src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.scss]: fixed

**Pull request merged**